cob_moveit_config/package.xml

```diff
@@ -0,0 +1,18 @@
+<package format="2">
+  <name>cob_moveit_config</name>
+  <version>0.6.6</version>
+  <description>
+     MoveIt config files for all cob and raw
+  </description>
+  <author email="mdl@ipa.fhg.de">Mathias Luedtke</author>
+  <maintainer email="mdl@ipa.fhg.de">Mathias Luedtke</maintainer>
+  <maintainer email="fxm@ipa.fhg.de">Felix Messmer</maintainer>
+
+  <license>BSD</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <exec_depend>moveit_setup_assistant</exec_depend>
```

Comment:
I don't like this dependency... That means that cob_bringup depends on moveit_core, moveit_ros_planning and moveit_ros_visualization... @ipa-mdl @ipa-fxm could be `cob_moveit_config/launch/setup_assistant.launch`  moved to **cob_moveit_bringup**??